src/config/server/config.server.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+  port: process.env.PORT || 3000,
```

Comment:
Don't use default export.
Also sometimes `||` operator as operator for default values doesn't work clearly. It would be better to use `??` instead.

Reply:
Still better to use `??`